libkbfs/tlf_handle_extension.go

```diff
@@ -23,14 +24,13 @@ const (
 	// TlfHandleExtensionNumberRegex is the regular expression matching the TlfHandleExtension
 	// number member in string form.
 	TlfHandleExtensionNumberRegex = "[0-9]+"
+	// TlfHandleExtensionUsernameRegex is the regular expression matching the TlfHandleExtension
+	// username member in string form.
+	TlfHandleExtensionUsernameRegex = "[a-z0-9_]+"
```

Comment:
For what it's worth, this looks like the username checker used by libkb: https://github.com/keybase/client/blob/master/go/libkb/checkers.go#L23

I guess it allows non-normalized names though.  What you have seems fine (though technically it should be at least two characters).